src/Samples/NetCoreServer/Properties/launchSettings.json

```diff
@@ -1,27 +1,12 @@
 {
-  "iisSettings": {
```

Comment:
I would be great to have an IIS sample with net.tcp binding

Reply:
That doesn't work yet. That depends TCP port sharing which needs client side named pipes support to tell SMSvcHost to listen on your behalf, and also needs WAS support to tell SMSvcHost to launch the IIS worker process.